File: output/gemma4-e4b/avatar/Java/Python/atcoder_ABC124_A.py
def solve():
    import sys
    # Read two integers from standard input
    try:
        line = sys.stdin.read().strip()
        if not line:
            return
        a, b = map(int, line.split())
    except EOFError:
        return
    except ValueError:
        # Handle case where input might not be two integers
        return

    sum_val = 0
    
    # The loop runs exactly twice (for i=0 and i=1)
    for _ in range(2):
        if a >= b:
            sum_val += a
            a -= 1
        else:
            sum_val += b
            b -= 1
            
    print(sum_val)

solve()

File: output/gemma4-e4b/avatar/Python/Java/atcoder_ABC124_A.java
import java.util.Scanner;

public class atcoder_ABC124_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read two integers a and b from standard input
        if (!scanner.hasNextInt()) {
            return;
        }
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        
        int c = 0;
        
        // Loop runs 2 times
        for (int i = 0; i < 2; i++) {
            // c += max(a, b)
            int maxVal = Math.max(a, b);
            c += maxVal;
            
            // Update a or b
            if (a >= b) {
                a -= 1;
            } else {
                b -= 1;
            }
        }
        
        System.out.println(c);
        
        scanner.close();
    }
}